src/detector.ts

```diff
@@ -1,15 +1,6 @@
-import { detectors } from './detectors'
-import { sources } from './sources'
-import {
-  BotdError,
-  BotDetectionResult,
-  BotDetectorInterface,
-  BotKind,
-  Component,
-  ComponentDict,
-  DetectionDict,
-  State,
-} from './types'
+import { BotDetectionResult, BotDetectorInterface, ComponentDict, DetectionDict } from './types'
+import { collect, detect } from './api'
+import { detectors, sources } from '.'
```

Comment:
This creates [a circular dependency](https://github.com/fingerprintjs/BotD/actions/runs/6145435367/job/16673913120#step:8:9)